src/CommunityToolkit/Xamarin.CommunityToolkit/Views/CameraView/Android/MotionEventHelper.android.cs

```diff
@@ -55,7 +54,7 @@ bool ShouldPassThroughElement()
 			}
 
 			// This is not a layout and it's transparent; the event can just pass through
-			if (element.InputTransparent)
+			if (element?.InputTransparent ?? false)
```

Comment:
element will not be null here, we did a check on the caller method.

Reply:
The earlier null check is verifying whether `element is Layout` and this null check is not within that scope